website/docs/legacy/09-interacting-with-starknet/05-deploy-account.md

```diff
@@ -0,0 +1,27 @@
+# Creating account
+
+## Overview
+
+In order to create a new account, you need to deploy an account contract. Starknet expects a certain interface from an account contract. [OpenZeppelin](https://github.com/OpenZeppelin/cairo-contracts/blob/main/src/openzeppelin/account/presets/Account.cairo) and [ArgentX](https://github.com/argentlabs/argent-contracts-starknet/blob/develop/contracts/account/ArgentAccount.cairo) provide account contract implementations that you can use.
+
+There are three ways of deploying a new account contract:
+- sending [`DEPLOY_ACCOUNT` transaction](https://github.com/starkware-libs/cairo-lang/releases/tag/v0.10.1)
+- using already deployed account contract and deploying new one via [the Universal Deployer Contract](https://community.starknet.io/t/universal-deployer-contract-proposal/1864)
+- (deprecated) sending [`DEPLOY` transaction](https://docs.starknet.io/documentation/develop/Blocks/transactions/#deploy_transaction)
```

Comment:
Does not exist anymore

Reply:
https://github.com/software-mansion/protostar/issues/1900